docs/user-guide/widgets.md

```diff
@@ -57,10 +57,82 @@ Just below the chart's preview, the following configurations are available:
 
 !!! Note
     The *No operation* option is used when the aggregation method is not needed for the chart. If *No Operation* is selected, no aggregation will be carried out for the chart and the WFS service will be used to generate the chart without using the WPS process `gs:Aggregate` in GeoServer.
+    
+    
+* Enable the chart's legend by activating **Display Legend**
 
-* Choose the **Color** (`Blue`, `Red`, `Green`, `Brown` or `Purple`) of the chart (or the **Color Ramp** for *Pie Charts*)
+* Choose the **Color** (`Blue`, `Red`, `Green`, `Brown` or `Purple`) of the chart (or the **Color Ramp** for *Pie Charts*) or choose to **Customize the color**. 
 
-* Enable the chart's legend by activating **Display Legend**
+#### Color customization
+
+For *Bar Charts* and *Pie Charts*, [MapStore](https://mapstore.geosolutionsgroup.com/mapstore/#/) provides the possibility to customize the colors of the charts bars and slices. From the **Color** option dropdown menu, the user can select the <img src="../img/button/custom-button.jpg" class="ms-docbutton"/> options and enter the **Custom Colors Settings** through the <img src="../img/button/change-media2.jpg" class="ms-docbutton"/> button.
+
+<img src="../img/widgets/custom_colors_settings.jpg" class="ms-docimage"/> 
+
+Through this section, the user is allowed to:
+
+* Choose the **Color** of the bars or the slides through the *Color Picker*.
+
+<img src="../img/widgets/custom_colors.gif" class="ms-docimage"/> 
+
+* Select an *Attribute* in the dropdown list as a **Classification attribute**.
+
+<img src="../img/widgets/classification-attribute.gif" class="ms-docimage"/> 
+
+Once the attribute is chosen, new options appear in the *Custom Color Settings* panel that allow the user to:
+
+* Enter a **Default Class Label** to be used in the legend for all values that will not be specifically classified in the following list.
+
+<img src="../img/widgets/default_class_label.jpg" class="ms-docimage"/> 
+
+!!! Note
+    The *Default Class Label* can be a **Custom Labels** with a '${legendValue}' used as placeholder for the Y Attribute.
```

Comment:
For both *Default Class Label* and  *Class Label* '${legendValue}' can be used as a placeholder for the Y Attribute (that can be further customized through the usual *Advanced Option*).